.travis.yml

```diff
@@ -39,7 +39,7 @@ before_script:
 #  - chmod +x travis-acceptance-tests.sh
 
 script: 
-  - ./gradlew uninstallAll createProdDebugCoverageReport #copyUnitTestBuildArtifacts
+  - ./gradlew uninstallAll createProdDebugAndroidTestCoverageReport #copyUnitTestBuildArtifacts
```

Comment:
As far as I can tell, such a task isn't provided for us, so I added some code to generate them for each application variant.